data/decks/c.viper.json

```diff
@@ -0,0 +1,188 @@
+{
+	"id": "c.viper",
+	"season": 3,
+	"display_name": "C. Viper",
+	"cardback": "cardback_defaultexceed.png",
+	"exceed_cost": 3,
+	"on_exceed": {
+		"effect_type": "draw",
+		"amount": 5
+		},
+	"ability_effects": [
+		{
+			"timing": "on_continuous_boost",
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "strike" },
+				{ "effect_type": "pass"}
+		]
+		},
+		{
+			"timing": "set_strike",
+			"condition": "is_not_critical",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+				"effect_type": "critical",
+				"character_effect": true
+			},
+			"gauge_max": 1
+		}
+	],
+	"exceed_ability_effects": [
+		{
+			"timing": "on_continuous_boost",
+			"effect_type": "set_used_character_bonus",
```

Comment:
Curious what this actually looks like in the UI.
Does it look right? Since pass doesn't actually give you this bonus.

Reply:
investigated this, it looks correct, only giving you crit when you set an attack